python/release/wheel/build_wheel_manylinux.sh

```diff
@@ -17,11 +17,16 @@ if [ $# -lt 3 ]; then
   exit 1
 fi
 
+ARCH=`uname -m`
```

Comment:
I suspect this file is now obsolete (I'm not 100% sure though): https://github.com/protocolbuffers/protobuf/pull/8250